public/js/loginValidation.js

```diff
@@ -1,47 +1,47 @@
-function isValidEmail(elementID) {
-    let inputVal = document.getElementById(elementID);
+let isValidEmail = (elementID) => {
+    let inputVal = document.getElementById(elementID)
+    let userEmailWarning = document.getElementById('userEmailWarning')
     let email = inputVal.value.trim();
-    let re = new RegExp("^\\w+([\\.-]?\\w+)*@\\w+([\\.-]?\\w+)*(\\.\\w{2,3})+$");
-
+    let re = new RegExp("^\\w+([\\.-]?\\w+)*@\\w+([\\.-]?\\w+)*(\\.\\w{2,3})+$")
+console.log(re.test(email))
     if (re.test(email)) {
-        inputVal.style.backgroundColor = "";
+        userEmailWarning.classList.remove('error')
         return true
     } else {
-        inputVal.style.backgroundColor = "pink";
+        userEmailWarning.classList.add('error')
         return false
     }
 }
 
-function postLoginDetails(inputs){
-    fetch("/api/login",
+let validateToPostInputs = () => {
```

Comment:
are we not just validating here? could we rename 'validateToPostInputs' to simply validateInputs ?

Reply:
renamed to 'validateInputs'